test/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriIQProviderTest.java

```diff
@@ -0,0 +1,190 @@
+                    "<source xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" ssrc=\"3703044675\">" +
+                      "<parameter value=\"5311ad66-bc71-7d4e-be71-66ac3a73182a\" name=\"cname\"/>" +
+                      "<parameter value=\"404a72b1-a51a-a545-a5e9-5a50cd94431c 135dd87a-1c7a-fb4d-9e42-d64cfbcfcc0d\" name=\"msid\"/>" +
+                      "<ssrc-info xmlns=\"http://jitsi.org/jitmeet\" owner=\"test@conference.brian2.jitsi.net/66e3ea10\"/>" +
+                    "</source>" +
+                    "<source xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" rid=\"1\">" +
+                      "<ssrc-info xmlns=\"http://jitsi.org/jitmeet\" owner=\"test@conference.brian2.jitsi.net/66e3ea10\"/>" +
+                    "</source>" +
+                    "<source xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" rid=\"2\">" +
+                      "<ssrc-info xmlns=\"http://jitsi.org/jitmeet\" owner=\"test@conference.brian2.jitsi.net/66e3ea10\"/>" +
+                    "</source>" +
+                    "<source xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" rid=\"3\">" +
+                      "<ssrc-info xmlns=\"http://jitsi.org/jitmeet\" owner=\"test@conference.brian2.jitsi.net/66e3ea10\"/>" +
+                    "</source>" +
+                    "<ssrc-group xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" semantics=\"SIM\">" +
+                      "<source ssrc=\"246878015\"/>" +
+                      "<source ssrc=\"3236476221\"/>" +
+                      "<source ssrc=\"3703044675\"/>" +
+                    "</ssrc-group>" +
+                    "<rid-group xmlns=\"urn:xmpp:jingle:apps:rtp:ssma:0\" semantics=\"SIM\">" +
+                      "<source rid=\"1\"/>" +
+                      "<source rid=\"2\"/>" +
+                      "<source rid=\"3\"/>" +
+                    "</rid-group>" +
+                  "</channel>" +
+                "</content>" +
+                "<channel-bundle id=\"66e3ea10\">" +
+                  "<transport xmlns=\"urn:xmpp:jingle:transports:ice-udp:1\" ufrag=\"117916df\" pwd=\"15929b4d44ae40fbcc6d51b6e4a468aa\">" +
+                    "<rtcp-mux/>" +
+                    "<fingerprint xmlns=\"urn:xmpp:jingle:apps:dtls:0\" hash=\"sha-256\" setup=\"active\" required=\"false\">44:AF:49:E3:3B:E0:0D:A2:FA:AB:F4:93:EC:5D:32:39:78:F8:01:06:1F:8E:E4:35:36:15:56:59:6B:3C:52:49</fingerprint>" +
+                  "</transport>" +
+                "</channel-bundle>" +
+              "</conference>" +
+            "</iq>";
+
+
+    XmlPullParserFactory xmlPullParserFactory;
+    XmlPullParser xmlPullParser;
+    ColibriIQProvider colibriIQProvider;
+
+
+    public void setUp()
+    {
+        try {
+            xmlPullParserFactory = XmlPullParserFactory.newInstance();
+            xmlPullParserFactory.setNamespaceAware(true);
+
+            xmlPullParser = xmlPullParserFactory.newPullParser();
+
+        } catch (XmlPullParserException e) {}
+        AbstractSmackInteroperabilityLayer.setImplementationClass(
```

Comment:
While we're at it, please move the braces to new lines (and the identation is wrong too).